explainer-server/app/models/ExplainerStore.scala

```diff
@@ -41,16 +38,33 @@ object ExplainerStore {
     ScanamoAsync.exec(dynamoDBClient)(operations).map(_.flatMap(_.toOption).get)
   }
 
-
   def update(id: String, fieldSymbol: Symbol, value: String): Future[Explainer] = {
-    assert(Set("headline","body").contains(fieldSymbol.name))
+    val allowed_fields = Set(
+      "headline",
+      "body",
+      "headline_published",
+      "body_published"
+    )
+    assert(allowed_fields.contains(fieldSymbol.name))
     val operations = for {
       _ <- explainersTable.update('id -> id, set(fieldSymbol -> value))
+      _ <- explainersTable.update('id -> id, set(Symbol("last_update_time_milli") -> (new DateTime).getMillis()))
       explainer <- explainersTable.get('id -> id)
     } yield {
       explainer
     }
     ScanamoAsync.exec(dynamoDBClient)(operations).map(_.flatMap(_.toOption).get)
   }
+
+  def create(): Future[Explainer] = {
+    val uuid = java.util.UUID.randomUUID.toString

```

Comment:
As discussed should not be a problem (it is safe to assume that a newly created UUID is different from any other UUID ever generated before).